libr/anal/sign.c

```diff
@@ -1493,6 +1493,10 @@ static void print_function_args_json(RAnal *a, PJ *pj, char *arg_type) {
 }
 
 static void print_type_json(RAnal *a, char *type, PJ *pj, size_t pos) {
+	if (pos == 0) {
```

Comment:
why doesn't it print anything if `pos == 0`? It seems weird. As said by trufae, please add a test for these changes.

Reply:
As stated:
> Skip the first type in the list. The first "type" in the list tells how many args there are and is technically not a type/arg itself.

This is redundant information to beavoided (the length of the json array should tell you how many there are)

Reply:
Ah yeah sorry, read it just now!

Reply:
np & thx for the review